lib/semian/circuit_breaker.rb

```diff
@@ -125,15 +125,14 @@ def disabled?
     end
 
     def maybe_with_half_open_resource_timeout(resource, &block)
-      result = nil
-
-      if half_open? && @half_open_resource_timeout && resource.respond_to?(:with_resource_timeout)
-        resource.with_resource_timeout(@half_open_resource_timeout) do
-          result = block.call
+      result =
```

Comment:
Small bikeshed, I can bring back the old syntax. I am used to assigning the value directly from the if, not sure what syntax you guys prefer.

Reply:
Just for context: The reason this kind of thing might frowned upon is because it causes a 7 line diff that we have to review manually to ensure that a typo size bug isn't added to the library.